fastq_filtrator.py

```diff
@@ -0,0 +1,148 @@
+            quality(input_fastq, yfilename, zfilename, leng_min, leng_max, gc_bounds_min, gc_bounds_max, quality_threshold, save_filtered)
+        elif menu == 5:
+            save(input_fastq, yfilename, zfilename, leng_min, leng_max, gc_bounds_min, gc_bounds_max, quality_threshold, save_filtered)
+        elif menu == 0:
+            exit()
+
+
+def start(input_fastq, yfilename, zfilename, leng_min, leng_max, gc_bounds_min, gc_bounds_max, quality_threshold, save_filtered):
+    print('Start')
+    print('fastq file name =', input_fastq)
+    print('sorted file name =', yfilename)
+    print('trash file name =', zfilename)
+    print('leng_min =', leng_min)
+    print('leng_max =', leng_max)
+    print('gc_bounds_min =', gc_bounds_min)
+    print('gc_bounds_max =', gc_bounds_max)
+    print('quality_threshold =', quality_threshold)
+    print('save_filtered =', save_filtered)
+    # pause = input("Press enter to continue ->")
+    x = open(input_fastq, 'r')  # открыть файл чтения
+    y = open(yfilename, 'w')    # создать файл для записи
+    y.close()
+    z = open(zfilename, 'w')    # создать файл для отфильтрованных
+    z.close()
+    all_fastq = x.readlines()
+    num_str = sum(1 for line in all_fastq)
+    print('num_str=', num_str)
+    for i in range(1, num_str+1, 4):
+        print('i=', i)
+        print(all_fastq[i], end='')
+        num_G = all_fastq[i].count('G')
+        num_C = all_fastq[i].count('G')
+        num_GC = num_G + num_C
+        len_str = len(all_fastq[i])
+        GC_content = num_GC / len_str * 100
+        print('len_str=', len_str)
+        print('GC_content=', GC_content)
+        quality = all_fastq[i+2]
+        print(quality)
+        sum_ord = 0
+        len_quality = len(quality)
+        for j in range(len_quality):  # подсчет качества
+            sum_ord = sum_ord + ord(quality[j])
+            quality_sum = sum_ord / len_quality
+        print('quality_sum=', quality_sum)
+        if len_str > leng_min and len_str < leng_max and GC_content > gc_bounds_min and GC_content < gc_bounds_max and quality_sum > quality_threshold:
+            y = open(yfilename, 'r+')
+            y.seek(0, 2)
+            y.write(all_fastq[i-1])
+            y.write(all_fastq[i])
+            y.write(all_fastq[i+1])
+            y.write(all_fastq[i+2])
+        elif save_filtered:
+            z = open(zfilename, 'r+')
+            z.seek(0, 2)
+            z.write(all_fastq[i - 1])
+            z.write(all_fastq[i])
+            z.write(all_fastq[i + 1])
+            z.write(all_fastq[i + 2])
+
+    prog(input_fastq, yfilename, zfilename, leng_min, leng_max, gc_bounds_min, gc_bounds_max, quality_threshold, save_filtered)
+
+
+def GC(input_fastq, yfilename, zfilename, leng_min, leng_max, gc_bounds_min, gc_bounds_max, quality_threshold, save_filtered):
+    print('gc_bounds_min =', gc_bounds_min)
+    print('gc_bounds_max =', gc_bounds_max)
+    e_check = str(input("Input the gc_bounds_min ->"))  # проверка на пустоту
```

Comment:
```suggestion
    e_check = input("Input the gc_bounds_min ->")  # проверка на пустоту
```
`input` уже возвращает строку, преобразование излишне. Комментарии лучше на английском